### PR TITLE
Add `devShell` example

### PR DESCRIPTION
**Pull request by atcol**
Adds a `nix develop` example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +79,35 @@ Here's an example of using it in nixos configuration.
 }
 ```
 
+### Example: A `devShell` Flake
+
+Assuming you have a valid `shell.nix`, running `nix develop` will create a shell with the default nightly Rust toolchain installed:
+
+```nix
+{
+  description = "Blah";
```
Comment (a-kenji) [inline README.md:88]: maybe write something like 'devShell example'

Comment (atcol) [reply README.md:88]: How about 
"Using `devShell` and `nix develop`"

?

Comment (a-kenji) [reply README.md:88]: Sounds much better!

Comment (atcol) [reply README.md:88]: Done! Thanks for reviewing

Comment (a-kenji) [inline README.md:88]: Sorry, I was talking about the blah, and not the header.

Comment (atcol) [reply README.md:88]: Haha what a silly my mistake

Comment (atcol) [reply README.md]: Fixed

Comment (a-kenji) [reply README.md]: Awesome!

File: README.md [hunk 2]
```diff
@@ -77,6 +79,65 @@ Here's an example of using it in nixos configuration.
...
+
+  inputs = {
+    nixpkgs.url      = "github:nixos/nixpkgs/nixos-unstable";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+    flake-utils.url  = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, rust-overlay, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+        let
```
Comment (oxalica) [inline README.md:98]: Indentation should be always 2 space.

File: README.md [hunk 3]
```diff
@@ -77,6 +79,65 @@ Here's an example of using it in nixos configuration.
...
+        let 
+          overlays = [ (import rust-overlay) ];
+          pkgs = import nixpkgs {
+            inherit system overlays;
+          };
+        in
+        {
+          devShell = pkgs.mkShell {
+             buildInputs = [
+              pkgs.openssl
```
Comment (oxalica) [inline README.md:107]: Indentation should be always 2 space.

**Review by oxalica — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (atcol) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (atcol) [reply README.md]: Fixed

File: README.md [hunk 4]
```diff
@@ -77,6 +79,48 @@ Here's an example of using it in nixos configuration.
...
+      let 
+        overlays = [ (import rust-overlay) ];
+        pkgs = import nixpkgs {
+          inherit system overlays;
+        };
+      in
+      {
+        devShell = pkgs.mkShell {
+        	buildInputs = [
+          	pkgs.openssl
```
Comment (oxalica) [inline README.md:107]: Please use spaces instead of tabs.

**Review by oxalica — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (atcol) [reply README.md]: Gvim on my windows box didn't have `expandtab` set! Fixed

**Review by oxalica — APPROVED** [review]

**Pull request merged**